vllm/model_executor/models/transformers.py

```diff
@@ -99,7 +101,7 @@ def replace_linear_class(
     vllm_linear_cls = {
         "colwise": ColumnParallelLinear,
         "rowwise": RowParallelLinear,
-    }.get(style)
+    }.get(style, ReplicatedLinear)
 
     if vllm_linear_cls is None:
         logger.warning(
```

Comment:
`vllm_linear_cls` can no longer be `None` so this warning will never be logged and can therefore be deleted